suite-common/icons/src/components/Icon.tsx

```diff
@@ -43,6 +48,10 @@ export const Icon = ({ name, customSize, size = 'large', color = 'iconDefault' }
 
     // Paint has to be Reanimated derived value, to allow color transition animation on the UI thread.
     const paint = useDerivedValue(() => {
+        // if the color is `undefined`, the `paint` is ignored and the svg source file
+        //  colors are used. This is useful for using multicolor icons.
+        if (areSvgColorsUsed || !color) return undefined;
```

Comment:
missing in deps

Reply:
the `svgSource` color prop used instead in the end.